consensus-types/epbs/execution_payload_envelope.go

```diff
@@ -0,0 +1,145 @@
+package epbs
```

Comment:
This entire file is untested. I understood it's used by someone else and we might have tests there